merge_single_band.py

```diff
@@ -0,0 +1,93 @@
+"""
+Merge matching bands from multiple geostationary satellites.
+
+Implements wavelength-based band matching across satellites as described in
+the MetNet Global paper. This is a minimal proof-of-concept focusing on the
+0.86 µm band from GK2A and GOES-East.
+
+TODO: Add proper reprojection and azimuth-based blending.
+"""
+
+import icechunk
+import numpy as np
+import xarray as xr
+
+
+def open_icechunk_store(prefix: str):
+    """Open an icechunk store from Source Cooperative."""
+    storage = icechunk.s3_storage(
+        bucket="bkr",
+        prefix=prefix,
+        region="us-east-1",
+        endpoint_url="https://data.source.coop",
+        anonymous=True,
+        force_path_style=True,
+    )
+    repo = icechunk.Repository.open(storage)
+    session = repo.readonly_session(branch="main")
+    return xr.open_zarr(session.store, consolidated=False)
+
+
+def get_band_data(ds, band_name: str, time_idx: int = 0):
+    """Extract a single timestep of a band."""
+    if band_name not in ds.data_vars:
+        return None
+
+    # Get single time slice
+    band = ds[band_name].isel(time=time_idx)
+
+    return band
+
+
+def merge_086um_band(time_idx: int = -1):
+    """
+    Merge 0.86 µm band from GK2A and GOES-East satellites.
+
+    Band matching:
+    - GK2A VI008: 0.863 µm
+    - GOES-East C03: 0.865 µm
+
+    Args:
+        time_idx: Time index to extract (-1 for most recent)
+
+    Returns:
+        Tuple of (merged_data, metadata)
+    """
+
+    # Load GK2A data
+    ds_gk2a = open_icechunk_store("geo/gk2a_1000m.icechunk")
```

Comment:
I think this is fine for this, but in general, I think allowing passing of a path to the store itself as an argument might be better. That way, if the data gets moved, or someone downloads a local copy, they can just pass the new path instead.